packages/core/src/interfaces/charts.ts

```diff
@@ -237,6 +242,20 @@ export interface PieChartOptions extends BaseChartOptions {
 	};
 }
 
+/**
+ * options specific to gauge charts
+ */
+export interface GaugeChartOptions extends PieChartOptions {
+	gauge?: {
+		arcWidth?: number;
+		fillColor?: string;
+		status?: Statuses;
```

Comment:
if status is set to `warning`, what does that do? maybe we need demos of those as well, also custom fill colors

Reply:
warning will make the arrow yellow - same statuses used by meter chart 
but ill add more demos for sure